packages/react-hooks/src/useForceUpdate.test.tsx

```diff
@@ -7,7 +7,7 @@ describe('useForceUpdate', () => {
     let renderCount = 0;
     const TestComponent: React.FunctionComponent = () => {
       const forceUpdate = useForceUpdate();
-      React.useEffect(() => forceUpdate(), []);
+      React.useEffect(() => forceUpdate(), [forceUpdate]);
```

Comment:
Another option is to silence this warning as obviously `forceUpdate` is a stable function

Reply:
forceUpdate actually isn't stable as checked in, though it obviously should be. I'm fixing this in ##14099.